hazelcast-sql/src/test/java/com/hazelcast/sql/SqlParameterTest.java

```diff
@@ -109,7 +109,9 @@ public void testParameters() {
                 + "CAST(? as DATE), "
                 + "CAST(? as TIME), "
                 + "CAST(? as TIMESTAMP), "
-                + "CAST(? as TIMESTAMP_WITH_TIME_ZONE), "
+                // TODO: change 'TIMESTAMP WITH LOCAL TIME ZONE' to 'TIMESTAMP WITH TIME ZONE'
```

Comment:
I'd remove these TODOs, the problem will pop up automatically when we remove the imdg engine...